content/rest-api/index.textile

```diff
@@ -17,11 +17,13 @@ jump_to:
   Push API:
     - register device#post-device-registration
     - subscribe to a channel#post-channel-subscription
+    - unregister device#delete-device-registration
+    - unsubscribe from channels#delete-channel-subscription
     - get registered device#get-device-registration
     - list registered devices#get-device-registrations
     - list channel subscriptions#get-channel-subscriptions
-    - unregister device#delete-device-registration
-    - unsubscribe from channel#delete-channel-subscription
+    - list channels#get-channels
```

Comment:
Given our recent conversation about range queries in Cassandra being difficult, would list channels not be a range query or do we store the list of channels registered in a bucket?